docs/.vuepress/components/TutorialList.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="tutorial-container">
+    <div v-for="(t, index) in tutorials" :key="t.title" class="tutorial-card">
+      <h3 class="tutorial-title">
+        <a :href="`${tutorialsRepo}${t.slug}/TUTORIAL.md`" target="_blank">{{ t.title }}</a>
+        <span> by {{ t.author }}</span>
+      </h3>
+      <div :class="`badge`" v-for="tag in t.tags" :key="tag" :style="{ backgroundColor: getColorForBadge(tag) }">{{ tag }}</div>
+      <p class="time"><span>⏳ Time:</span> <strong>{{ t.time }}</strong></p>
+      <p>{{ t.description }}</p>
+      <p>
+        <a class="button" :href="`${tutorialsRepo}${t.slug}/TUTORIAL.md`" target="_blank">Tutorial repository</a>
+      </p>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted } from "vue";
+// const tutorialsRepo = "https://raw.githubusercontent.com/zkSync-Community-Hub/tutorials/main/";
+const tutorialsRepo = "https://raw.githubusercontent.com/zkSync-Community-Hub/tutorials/feat/json-descriptions/";
```

Comment:
Should we merge this into `main` before merging this PR?

Reply:
Yes, agreed! Here's the PR: https://github.com/zkSync-Community-Hub/tutorials/pull/15
Then I can edit this part of the code to point to main and before merging test it works well

Reply:
Copy. I've approved/merged the other PR